pkg/report/linux.go

```diff
@@ -661,3 +711,93 @@ var linuxOopses = []*oops{
 		[]*regexp.Regexp{},
 	},
 }
+
+func isCorrupted(desc string, text string) bool {
+	if !strings.Contains(text, "Call Trace") && !strings.Contains(text, "backtrace") {
+		// Text must contain 'Call Trace' or 'backtrace'.
+		return true
+	}
+	corruptedDescRegExps := []*regexp.Regexp{
+		// 'kernel panic: Fatal exception' is usually printed after BUG,
+		// so if we captured it as a report description, that means the
+		// report got truncated and we missed the actual BUG header.
+		regexp.MustCompile(`kernel panic: Fatal exception`),
+		// Same, but for WARNINGs and KASAN reports.
+		regexp.MustCompile(`kernel panic: panic_on_warn set`),
+		// Sometimes timestamps get merged into the middle of report description.
+		regexp.MustCompile(`\[ *[0-9]+\.[0-9]+\]`),
+		regexp.MustCompile(`\[ *[0-9]+\.NUM\]`),
+		regexp.MustCompile(`\[ *NUM\.NUM\]`),
```

Comment:
I think we should run isCorrupted _before_ we introduce these NUMs. If we run isCorrupted after, we have these additional possibilities and we can't distinguish if NUM/ADDR was in output or it's we introduced it.

Reply:
Done in #427.